testing/test_deploy_app.py

```diff
@@ -33,7 +39,7 @@ def may_get_env_var(name):
   else:
     raise Exception("%s not set" % name)
 
-def make_deploy_call(args):
+def prepare_request_data(args):
```

Comment:
Why does the prototype need to load gcp_prototype.yaml? Shouldn't that be handled by the backend?